templates/react-native/src/index.ts.twig

```diff
@@ -0,0 +1,12 @@
+export { Client, Query, {{spec.title | caseUcfirst}}Exception } from './client';
+{% for service in spec.services %}
+export { {{service.name | caseUcfirst}} } from './services/{{service.name | caseDash}}';
+{% endfor %}
+export type { Models, Payload, RealtimeResponseEvent, UploadProgress } from './client';
+export type { QueryTypes, QueryTypesList } from './query';
```

Comment:
Let's export query from here instead of client

Reply:
updated